tests/unit/service_test.py

```diff
@@ -99,14 +99,12 @@ def test_get_volumes_from_service_no_container(self):
 
     def test_split_domainname_none(self):
         service = Service('foo', image='foo', hostname='name', client=self.mock_client)
-        self.mock_client.containers.return_value = []
```

Comment:
Nice. Sorry to be picky, but would you mind doing this cleanup (and the `__` -> `_` changes) in a separate commit?